testResources/exampleJavaPipeline.jenkins

```diff
@@ -8,4 +8,5 @@ def app = "app"
 
 withPipeline('java', product, app) {
 
+  enableHighLevelDataSetup()
```

Comment:
I don't think the default test pipeline should have a feature enabled like this,

Could you create a separate test for it?

Reply:
I have reverted this from default pipeline. The related test case will be created once we add the new pipeline type for highLevelDataSetup.

Reply:
resolved in [b7ebe7a5e40a9e22fc3300f30a00582f0183cfb1](https://github.com/hmcts/cnp-jenkins-library/pull/747/commits/b7ebe7a5e40a9e22fc3300f30a00582f0183cfb1)